### ISSUE TITLE
Ignore files processing doesn't follow established ignore rules

### ISSUE THREAD

**Issue by cehoffman** [app-registry/appr#83]
I have the following in a `.helmignore`

~~~
/.git
/.gitlab-ci.yml
test/
appr_charts/
~~~

This doesn't actually ignore anything and is a divergence from the way helm processes `.helmignore`.

See the test cases for helm [here](https://github.com/kubernetes/helm/blob/2ed70fc21d5ab3584968846645e954b4f7b7ae4e/pkg/ignore/rules_test.go#L92-L121)

This syntax is also valid for other ignore formats. I think the current code wrongly includes the path to the current directory from the drive root.

**Comment by ant31** [comment]
what should be ignored by: 
```
/.git
/.gitlab-ci.yml
```
is that absolute path or relative path ?

**Comment by cehoffman** [comment]
This is syntax for path from project root directory. So a mixture of relative (to project), and absolute within the project.

**Comment by ant31** [comment]
it should be fixed in #84, I'll release it in 0.7.4

**Comment by cehoffman** [comment]
Awesome, thanks for the quick implementation.
